modules/install/join_k3s_agent.sh

```diff
@@ -82,16 +80,33 @@ disable_cloud_setup() {
   fi
 }
 
-install(){
-  export "$install_mode"="$version"
+export_variables() {
+    export "$install_mode"="$version"
+}
 
+install_k3s(){
   if [[ -n "$channel"  ]]; then
     curl -sfL https://get.k3s.io | INSTALL_K3S_CHANNEL=$channel sh -s - agent
   else
     curl -sfL https://get.k3s.io | sh -s - agent
   fi
-  sleep 15
+}
+
+check_service() {
+  if systemctl is-active --quiet k3s-agent; then
+      printf "K3s is running on agent node ip: %s\n" "$public_ip"
+  else
+      printf "K3s failed to start on agent node ip %s\n" "$public_ip"
```

Comment:
Can we keep this message consistent? In some cases we are using `joining` and in some we are not.